.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,22 @@
+name: ansible-lint
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
```

Comment:
Does this mean we only lint on a pull request for these files? Either way, I don't think these are necessary.

Reply:
Thanks. Done.